cuqi/samples/_samples.py

```diff
@@ -399,17 +442,19 @@ def plot_ci(self,percent=95,exact=None,*args,plot_envelope_kwargs={},**kwargs):
             self.geometry.plot(lo_conf)
             plt.title("Lower credibility interval limit")
         else:
-            lci = self.geometry.plot_envelope(lo_conf, up_conf,color='dodgerblue',**pe_kwargs)
+            lci = self.geometry.plot_envelope(
+                lo_conf, up_conf, color='dodgerblue', **pe_kwargs)
             
-            lmn = self.geometry.plot(mean,*args,**kwargs)
-            if exact is not None: #TODO: Allow exact to be defined in different space than mean?
+            lmn = self.geometry.plot(mean, *args, **kwargs)
+            if exact is not None:  #TODO: Allow exact to be defined in different space than mean?
                 if isinstance(exact, CUQIarray):
-                    lex = exact.plot(*args,**kwargs)
+                    lex = exact.plot(*args, **kwargs)
```

Comment:
Should this check for plot_par / is_par? We would want the exact to be expressed on the same space

Reply:
There are checks for is_par and plot_par and setting them automatically if needed in this method (let me know if this does not solve the problem you are thinking about). One thing I was thinking about is removing the args from `lex = exact.plot(*args, **kwargs)` and from plot_ci signature. I think is is confusion since we have two plotting types here, plot envelope and the regular plot.

Reply:
Got it. I missed adding the plot_par to kwargs